ext/drx/drx.h

```diff
@@ -228,6 +228,34 @@ DR_EXPORT
 bool
 drx_register_soft_kills(bool (*event_cb)(process_id_t pid, int exit_code));
 
+/***************************************************************************
+ * INSTRUCTION LIST
+ */
+
+DR_EXPORT
+/**
+ * Returns the number of instructions (including meta-instructions) inside a basic block
+ * \p bb.
```

Comment:
Update the parameter name and what it is, and the "bb" below.  Optional: I suppose we could use actual doxygen @param markup and then doxygen would tell us when the name is wrong.

Reply:
Done.